introduction.tex

```diff
@@ -1,2 +1,40 @@
+SymPy is an full featured computer algebra system (CAS) written in the Python
+programming language. It is open source, licensed under the extremely
+permissive BSD license, which allows anyone to reuse SymPy or its source code,
+even for commercial purposes. SymPy was started by Ond\v{r}ej \v{C}ert\'{\i}k
+in 2005, and it has since grown into a large open source project, with over
+500 contributors.
+% citation?
 
+SymPy is written entirely in the Python programming language,
```

Comment:
Yes, in fact we already started to work on a paper with @thilinarmtb and @isuruf. After the sympy paper is submitted, we will start a paper on symengine, and we will follow the exact same workflow (open on github, authorship criteria, etc.). Right now I want to concentrate on getting the sympy paper out.